static/src/javascripts/projects/common/modules/commercial/dfp/private/post-message.js

```diff
@@ -0,0 +1,7 @@
+define(function () {
+    return postMessage;
+
+    function postMessage(message, targetWindow) {
+        targetWindow.postMessage(JSON.stringify(message), location.protocol + '//tpc.googlesyndication.com');
```

Comment:
I would reference `location.protocol + '//tpc.googlesyndication.com'` as a shared definition, so that the `postMessage` function in the `post-message` module and the `onMessage` function in `messenger` cannot deviate. It is crucial that they match.